### DISCUSSION TITLE
[PHP] Extract / generate interface from existing class php?

### DISCUSSION THREAD

**Discussion by DaddyDomi** [apache/netbeans#5310] [Q&A]
Is there a functionality to extract / generate interface based on existing PHP class (in Netbeans or using external tools)? Or is it possible to add such functionality to Netbeans?

**Comment by KacerCZ** [comment]
NetBeans currently does not support this kind of refactoring.

**Reply by DaddyDomi** [reply]
Do you know any external tools, than can be used for it?
